.github/workflows/build-mageos-nightly.yml

```diff
@@ -0,0 +1,45 @@
+name: Build & check Mage-OS Nightly
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '20 11 * * *'
+  workflow_dispatch:
+    inputs:
+      composer_cache_dir:
```

Comment:
This is unused?

Reply:
The composer_cache_dir is passed as an input to `graycoreio/github-actions-magento2/.github/workflows/integration.yaml@main`

Reply:
See the last line:
```
  integration-check:
    needs: [deploy, compute-nightly-service-versions]
    uses: graycoreio/github-actions-magento2/.github/workflows/integration.yaml@main
    with:
      package_name: mage-os/demo-package
      use_local_source: false
      matrix: ${{ needs.compute-nightly-service-versions.outputs.matrix }}
      test_command: ../../../vendor/bin/phpunit ../../../vendor/mage-os/demo-package/Test/Integration
      fail-fast: false
      magento_repository: https://nightly.mage-os.org
      composer_cache_key: ${{ inputs.composer_cache_key || 'v1' }}
```